### ISSUE TITLE
Amplify Github Integration not working

### ISSUE THREAD

**Issue by yeezick** [aws-amplify/amplify-hosting#3019]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Region

us-east-1

### Amplify Hosting feature

Not applicable

### Describe the bug

Hello, it seems that the extensions for amplify in github integrations is down. A few team members and myself have tried to go through the [tutorial to deploy a react app on amplify](https://aws.amazon.com/getting-started/hands-on/build-react-app-amplify-graphql/module-one/) but can't get past the "Deploy your app with AWS Amplify" step. After logging into my GitHub, it attempts to install the amplify-us-east-1 app on our GitHub but GitHub returns a 502 for this extension. 

[Tutorial](https://aws.amazon.com/getting-started/hands-on/build-react-app-amplify-graphql/module-one/)
[Installation url](https://github.com/apps/aws-amplify-us-east-1/installations/new)

To replicate my steps, you can follow the tutorial above and will be met with my issue in the first module. 

### Expected behavior

Amplify should be able to install the `aws-amplify-us-east-1` integration on my github. 

### Reproduction steps

https://aws.amazon.com/getting-started/hands-on/build-react-app-amplify-graphql/module-one/

The bug will be uncovered within the first module. In short:
- initialize a repo on github
- initialize a react app within the repo
- log into aws amplify console 
- attempt to deploy the repo with amplify hosting

**Comment by hloriana** [comment]
Hi @yeezick 👋🏽 I'm unable to reproduce this behavior, I was able to successfully install the app and host my application. Can you please provide some screenshots or a gif of what you seen as you try to install the app? That could help us to understand the problem since we cannot reproduce this behavior on our end.

**Comment by yeezick** [comment]
Hi @hloriana thanks for the quick response. i wasn't sure whether it was a github or aws error so I made tickets for both and this is what they responded:

"Thanks for reporting this --we have received similar reports and there is an internal issue with the engineering team where it is being tracked. From the investigation so far, it appears users affected are outside collaborators in orgs with many repositories -- in your case me50 organization.

Would it be possible to remove yourself as a collaborator in this organization and see if the issue is resolved for you?"

This was weird since I joined me50 over a year ago and was able to complete the tutorial at least twice on my own (within last month) before going through it again yesterday. 

After removing the me50 organization, it worked for me.
